tests/test_astar.py

```diff
@@ -0,0 +1,118 @@
+import sys
+sys.path.append('../')
```

Comment:
If you are using conda, you can `conda develop /path/to/two_d_nav_gym`. If you are using pip, you can add a `setup.py` to `/path/to/two_d_nav_gym`, and use `pip install -e /path/to/two_d_nav_gym`. Then this path will be appended to the python path. 

sys.path.append is not the common practice.

Reply:
resolved